pkg/controller/cronjob/utils.go

```diff
@@ -106,9 +106,13 @@ func mostRecentScheduleTime(cj *batchv1.CronJob, now time.Time, schedule cron.Sc
 	if timeBetweenTwoSchedules < 1 {
 		return earliestTime, nil, 0, fmt.Errorf("time difference between two schedules is less than 1 second")
 	}
-	timeElapsed := int64(now.Sub(t1).Seconds())
-	numberOfMissedSchedules := (timeElapsed / timeBetweenTwoSchedules) + 1
-	mostRecentTime := time.Unix(t1.Unix()+((numberOfMissedSchedules-1)*timeBetweenTwoSchedules), 0).UTC()
+
+	var numberOfMissedSchedules int64
```

Comment:
They are zero valued by default right, any specific reason we want to initialize them explicitly because if i initialize i need to do it to a zero value anyway?